Testscripts/Linux/CORE-InitrdModulesCheck.sh

```diff
@@ -67,22 +67,17 @@ fi
 
 # Rebuild array to exclude built-in modules
 skip_modules=()
-
-vmbusIncluded=$(grep CONFIG_HYPERV=y /boot/config-$(uname -r))
-if [ "$vmbusIncluded" ]; then
-    skip_modules+=("hv_vmbus.ko")
-    LogMsg "hv_vmbus module is built-in. Skipping module. "
-fi
-storvscIncluded=$(grep CONFIG_HYPERV_STORAGE=y /boot/config-$(uname -r))
-if [ "$storvscIncluded" ]; then
-    skip_modules+=("hv_storvsc.ko")
-    LogMsg "hv_storvsc module is built-in. Skipping module. "
-fi
-netvscIncluded=$(grep CONFIG_HYPERV_NET=y /boot/config-$(uname -r))
-if [ "$netvscIncluded" ]; then
-    skip_modules+=("hv_netvsc.ko")
-    LogMsg "hv_netvsc module is built-in. Skipping module. "
-fi
+config_path="/boot/config-$(uname -r)"
+declare -A config_modulesDic
+config_modulesDic=([CONFIG_HYPERV=y]="hv_vmbus.ko" [CONFIG_HYPERV_STORAGE=y]="hv_storvsc.ko" [CONFIG_HYPERV_NET=y]="hv_netvsc.ko")
```

Comment:
this looks good.
the only suggestion for better formatting might be to put each module on a new line, like:
config_modulesDic=(
module=x
module=y
)